### ISSUE TITLE
Add support for other job schedulers

### ISSUE THREAD

**Issue by forsyth2** [E3SM-Project/zppy#101]
Currently `zppy` only supports SLURM. This means that `zppy` won't work on machines using the Cobalt job scheduler for example. So, we should add support for other schedulers.

**Comment by forsyth2** [comment]
@golaz Is this ok to wait for v1.1.0 or should we try to get this into v1.0.0? I don't imagine the implementation being trivial.

**Comment by golaz** [comment]
@forsyth2 : yes, this can wait until later and there is a well defined need.

**Comment by forsyth2** [comment]
Closing since we don't have a well-defined need for this.
